liquid_node/nomad.py

```diff
@@ -21,12 +23,34 @@ def name(check):
             )
             return check['Name']
 
-        spec = self.post(f'jobs/parse', {'JobHCL': hcl})
         for group in spec['TaskGroups'] or []:
             for task in group['Tasks'] or []:
                 for service in task['Services'] or []:
                     yield service['Name'], [name(check) for check in service['Checks'] or []]
 
+    def get_resources(self, spec):
+        """Generates (task, type, resources) tuples with resource stranzas for
+        the supplied job, with default values set for memory, disk and cpu."""
+
+        DEFAULTS = {
+            'MemoryMB': 300,
+            'CPU': 100,
+            'DiskMB': 300,
+        }
+
+        def set_defaults(res):
+            if res is None:
+                res = {}
+            for key in DEFAULTS:
+                if not res.get(key):
+                    res[key] = DEFAULTS[key]
+            return res
+
+        for group in spec['TaskGroups'] or []:
+            for task in group['Tasks'] or []:
+                name = f'{spec["Name"]}-{group["Name"]}-{task["Name"]}'
+                yield name, spec['Type'], set_defaults(task['Resources'])
```

Comment:
`set_defaults(task['Resources'])` is like `resources = dict(DEFAULTS); resources.update(task['Resources'] or {})`